libs/minecraft/ColorCodes.arucas

```diff
@@ -0,0 +1,565 @@
+	RED1(255,0,0),
+	RED2(238,0,0),
+	RED3(205,0,0),
+	RED4(139,0,0),
+	ROSYBROWN(188,143,143),
+	ROSYBROWN1(255,193,193),
+	ROSYBROWN2(238,180,180),
+	ROSYBROWN3(205,155,155),
+	ROSYBROWN4(139,105,105),
+	ROYALBLUE(65,105,225),
+	ROYALBLUE1(72,118,255),
+	ROYALBLUE2(67,110,238),
+	ROYALBLUE3(58,95,205),
+	ROYALBLUE4(39,64,139),
+	SALMON(250,128,114),
+	SALMON1(255,140,105),
+	SALMON2(238,130,98),
+	SALMON3(205,112,84),
+	SALMON4(139,76,57),
+	SANDYBROWN(244,164,96),
+	SAPGREEN(48,128,20),
+	SEAGREEN1(84,255,159),
+	SEAGREEN2(78,238,148),
+	SEAGREEN3(67,205,128),
+	SEAGREEN4(46,139,87),
+	SEASHELL1(255,245,238),
+	SEASHELL2(238,229,222),
+	SEASHELL3(205,197,191),
+	SEASHELL4(139,134,130),
+	SEPIA(94,38,18),
+	SGIBEET(142,56,142),
+	SGIBRIGHTGRAY(197,193,170),
+	SGICHARTREUSE(113,198,113),
+	SGIDARKGRAY(85,85,85),
+	SGIGRAY12(30,30,30),
+	SGIGRAY16(40,40,40),
+	SGIGRAY32(81,81,81),
+	SGIGRAY36(91,91,91),
+	SGIGRAY52(132,132,132),
+	SGIGRAY56(142,142,142),
+	SGIGRAY72(183,183,183),
+	SGIGRAY76(193,193,193),
+	SGIGRAY92(234,234,234),
+	SGIGRAY96(244,244,244),
+	SGILIGHTBLUE(125,158,192),
+	SGILIGHTGRAY(170,170,170),
+	SGIOLIVEDRAB(142,142,56),
+	SGISALMON(198,113,113),
+	SGISLATEBLUE(113,113,198),
+	SGITEAL(56,142,142),
+	SIENNA(160,82,45),
+	SIENNA1(255,130,71),
+	SIENNA2(238,121,66),
+	SIENNA3(205,104,57),
+	SIENNA4(139,71,38),
+	SILVER(192,192,192),
+	SKYBLUE(135,206,235),
+	SKYBLUE1(135,206,255),
+	SKYBLUE2(126,192,238),
+	SKYBLUE3(108,166,205),
+	SKYBLUE4(74,112,139),
+	SLATEBLUE(106,90,205),
+	SLATEBLUE1(131,111,255),
+	SLATEBLUE2(122,103,238),
+	SLATEBLUE3(105,89,205),
+	SLATEBLUE4(71,60,139),
+	SLATEGRAY(112,128,144),
+	SLATEGRAY1(198,226,255),
+	SLATEGRAY2(185,211,238),
+	SLATEGRAY3(159,182,205),
+	SLATEGRAY4(108,123,139),
+	SNOW1(255,250,250),
+	SNOW2(238,233,233),
+	SNOW3(205,201,201),
+	SNOW4(139,137,137),
+	SPRINGGREEN(0,255,127),
+	SPRINGGREEN1(0,238,118),
+	SPRINGGREEN2(0,205,102),
+	SPRINGGREEN3(0,139,69),
+	STEELBLUE(70,130,180),
+	STEELBLUE1(99,184,255),
+	STEELBLUE2(92,172,238),
+	STEELBLUE3(79,148,205),
+	STEELBLUE4(54,100,139),
+	TAN(210,180,140),
+	TAN1(255,165,79),
+	TAN2(238,154,73),
+	TAN3(205,133,63),
+	TAN4(139,90,43),
+	TEAL(0,128,128),
+	THISTLE(216,191,216),
+	THISTLE1(255,225,255),
+	THISTLE2(238,210,238),
+	THISTLE3(205,181,205),
+	THISTLE4(139,123,139),
+	TOMATO1(255,99,71),
+	TOMATO2(238,92,66),
+	TOMATO3(205,79,57),
+	TOMATO4(139,54,38),
+	TURQUOISE(64,224,208),
+	TURQUOISE1(0,245,255),
+	TURQUOISE2(0,229,238),
+	TURQUOISE3(0,197,205),
+	TURQUOISE4(0,134,139),
+	TURQUOISEBLUE(0,199,140),
+	VIOLET(238,130,238),
+	VIOLETRED(208,32,144),
+	VIOLETRED1(255,62,150),
+	VIOLETRED2(238,58,140),
+	VIOLETRED3(205,50,120),
+	VIOLETRED4(139,34,82),
+	WARMGREY(128,128,105),
+	WHEAT(245,222,179),
+	WHEAT1(255,231,186),
+	WHEAT2(238,216,174),
+	WHEAT3(205,186,150),
+	WHEAT4(139,126,102),
+	WHITE(255,255,255),
+	WHITESMOKE(245,245,245),
+	YELLOW1(255,255,0),
+	YELLOW2(238,238,0),
+	YELLOW3(205,205,0),
+	YELLOW4(139,139,0);
```

Comment:
Do you really need to list every possible colour under the sun here?
I think that those that have multiple variants can be yeeted. e.g. for `YELLOW` I would remove `YELLOW2, YELLOW3, YELLOW4`. Less is more, this is just super confusing, you will never need this amount of colours... ever.

Reply:
The painful thing is it has different value of brightness, but its all yellow since R:G ratio is 1, I'm not really sure how could someone handle this....
Well I guess the real problem is having "1" at the end of common color names